roles/infra/roles/storages/tasks/main.yml

```diff
@@ -77,7 +77,7 @@
     - storages
     - storage_connections
 
-- name: Validate connections of storages
+- name: Update storage pamareters
```

Comment:
pamareters --> parameters
:)

Reply:
Fixed in: https://github.com/oVirt/ovirt-ansible-collection/pull/282